src/Resources/app/storefront/src/js/filter-range.plugin.js

```diff
@@ -0,0 +1,98 @@
+import FilterRangePlugin from 'src/plugin/listing/filter-range.plugin';
+import '../../node_modules/nouislider/distribute/nouislider.css';
+import deepmerge from 'deepmerge';
+import DomAccess from 'src/helper/dom-access.helper';
+
+const noUiSlider = require('@nouislider');
+
+export default class FlFilterRangePlugin extends FilterRangePlugin {
+
+  static options = deepmerge(FilterRangePlugin.options, {
+    sliderContainer: '.fl--range-slider',
+  });
+
+  _init() {
+    super._init();
+    this._inputMin = DomAccess.querySelector(this.el, this.options.inputMinSelector);
+    this._inputMax = DomAccess.querySelector(this.el, this.options.inputMaxSelector);
+
+    const slider = document.createElement('div');
+    this._sliderContainer = DomAccess.querySelector(this.el, this.options.sliderContainer);
+    this._sliderContainer.prepend(slider);
+
+    let start = this._inputMin.value ? this._inputMin.value : this.options.price.min;
```

Comment:
The range slider does not seem to work on other filters except price.

![image](https://user-images.githubusercontent.com/20555112/109942378-6dae9100-7cd4-11eb-96ae-1b9b9c22b529.png)

I have created a custom field and assigned some products numerical values to it. Re-importing and configuring this filter as a range-slider, will return the proper response:

![image](https://user-images.githubusercontent.com/20555112/109942511-8f0f7d00-7cd4-11eb-9431-25973f4e9b17.png)